### PR TITLE
fix: buttons view on mobile

### PR DESCRIPTION
**Pull request by ron-huberfeld**
Add @media query which hides 
- exercise id
- notebook id
- number of errors
and reduce exercise name font size.

### CODE REVIEW

File: lms/static/my.css [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+    display: none !important;
+  }
+  .which-notebook {
+    display: none !important;
+  }
+  .badge {
+    display: none !important;
+  }
+  .ex-title {
+    font-size: 16px !important;
```
Comment (orronai) [inline lms/static/my.css:12]: I think it's better to use rem units

File: lms/static/my.css [hunk 2]
```diff
@@ -1,3 +1,18 @@
+@media screen and (max-width: 30em) {
```
Comment (yammesicka) [inline lms/static/my.css:1]: Specifically here, I would probably use `px`. (~768px)

File: lms/static/my.css [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+  .exercise-id {
+    display: none !important;
```
Comment (yammesicka) [inline lms/static/my.css:3]: Try not to use `!important`. If needed, move `@media` queries to the end of the CSS document.

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: lms/static/my.css (hunk 2 above)
Comment (ron-huberfeld) [reply lms/static/my.css:1]: Can you explain why so big viewport to start removing items from the view?

Comment (ron-huberfeld) [reply lms/static/my.css]: Ignore my question. I modified the code accordingly.

**Review by yammesicka — APPROVED** [review]

**Pull request merged**